addons/l10n_ec/models/account_tax.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+
+    l10n_ec_code_base = fields.Char(string='Code base', help='Tax declaration code of the base amount prior to the calculation of the tax')
+    l10n_ec_code_applied = fields.Char(string='Code applied', help='Tax declaration code of the resulting amount after the calculation of the tax')
+    l10n_ec_code_ats = fields.Char(string='Code ATS', help='Tax Identification Code for the Simplified Transactional Annex')

```

Comment:
Ok, in a new module, this is for the ATS report.

Reply:
I find it difficult to move to a higher module as the taxes are created from the tax.teemplate only at the moment of installation of l10n_ec module. If it is not loaded at the moment of installation of l10n_ec it will be really a problem to map them later. 

The fields are not mandatory and besides the need to update the module might not pose any other problem to existing instances.